src/MoBi.R/MinimalImplementations/SimulationDiagramManager.cs

```diff
@@ -0,0 +1,84 @@
+﻿using System;
+using System.Drawing;
+using MoBi.Core.Domain.Model;
+using MoBi.Core.Domain.Model.Diagram;
+using OSPSuite.Core.Diagram;
+using OSPSuite.Core.Domain;
+
+namespace MoBi.R.MinimalImplementations
+{
+   public class SimulationDiagramManager : ISimulationDiagramManager
```

Comment:
Where is all this DiagramStuff used/required? Can't we modify the code instead? It looks like so much extra work for presentation stuff